### PR TITLE
[11.0][MIG] stock_warehouse_orderpoint_stock_info_unreserved

### PR DESCRIPTION
**Pull request by mpanarin**
depends on 
- [x] https://github.com/OCA/stock-logistics-warehouse/pull/375

### CODE REVIEW

File: stock_warehouse_orderpoint_stock_info_unreserved/models/stock_warehouse_orderpoint.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        string='Quantity On Location (Unreserved)',
+        compute='_compute_product_available_qty',
+    )
+
+    @api.multi
+    def _compute_product_available_qty(self):
+        super()._compute_product_available_qty()
+        op_by_loc = defaultdict(lambda: self.env['stock.warehouse.orderpoint'])
+        for order in self:
+            op_by_loc[order.location_id] |= order
```
Comment (guewen) [inline stock_warehouse_orderpoint_stock_info_unreserved/models/stock_warehouse_orderpoint.py]: This pattern of extending the recordset is very slow compared to using a true `set`, then create a new recordset out of the final set. See https://github.com/OCA/connector-prestashop/pull/53/commits/5d617fdbc85b1fdb23a4557b8d456b1b2e43720a

**Review by guewen — CHANGES_REQUESTED** [review]

**Review by guewen — APPROVED** [review]

File: stock_warehouse_orderpoint_stock_info_unreserved/tests/test_stock_warehouse_orderpoint.py [hunk 2]
```diff
@@ -1,61 +1,63 @@
...
+from odoo.tests.common import SavepointCase
 
 
-class TestStockWarehouseOrderpoint(TransactionCase):
-
-    def setUp(self):
-        super(TestStockWarehouseOrderpoint, self).setUp()
+class TestStockWarehouseOrderpoint(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
```
Comment (asaunier) [inline stock_warehouse_orderpoint_stock_info_unreserved/tests/test_stock_warehouse_orderpoint.py]: Just a candid question: what's the point of renaming ``self`` to ``cls``?

Edit: OK, it's a pep8 recommendation, see https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments
> Always use self for the first argument to instance methods.
> Always use cls for the first argument to class methods.

**Review by asaunier — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Dependency is merged. Please rebase.

**Comment by gurneyalex** [comment]
@mpanarin can you rebase?

**Review by gurneyalex — APPROVED** [review]

**Review by bodedra — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
Tested, LGTM :+1:

**Pull request merged**